core/sms_server.py

```
@@ -0,0 +1,1305 @@
+import os
+import json
+import asyncio
+import logging
+import secrets
+import hashlib
+import uuid as uuid_module
+import re
+from datetime import datetime, timezone
+from typing import List, Dict, Optional
```

Ruff (0.14.5)

`typing.List` is deprecated, use `list` instead

(UP035)